keywords.txt

```diff
@@ -2,6 +2,7 @@
 # Datatypes (KEYWORD1)
 #######################################
 MyMessage	KEYWORD1
+MyBlob		KEYWORD1
```

Comment:
Between `MyBlob` and `KEYWORD1 `**must** be a **single** tab char but not two. Otherwise, highlighting of this keyword in the Arduino IDE will not work.

Reply:
My Arduino Studio highlights the keword fine though, but I have updated it.